{{cookiecutter.module_name}}/Makefile

```diff
@@ -100,18 +100,22 @@ json/format: | guard/program/jq
 	$(FIND_JSON) | $(XARGS) bash -c 'echo "$$(jq --indent 4 -S . "{}")" > "{}"'
 	@ echo "[$@]: Successfully formatted JSON files!"
 
-docs/%: README_PARTS := _docs/MAIN.md <(echo) <(terraform-docs markdown table .)
-docs/%: README_FILE ?= README.md
-
-docs/lint: | guard/program/terraform-docs
-	@ echo "[$@]: Linting documentation files.."
-	diff $(README_FILE) <(cat $(README_PARTS))
-	@ echo "[$@]: Documentation files PASSED lint test!"
-
-docs/generate: | guard/program/terraform-docs
+guard/tfdocs-awk:
+	mkdir -p scripts
+	$(CURL) -o scripts/terraform-docs.awk https://raw.githubusercontent.com/plus3it/tfdocs-awk/master/terraform-docs.awk
```

Comment:
Perhaps just curl the repo tar.gz archive and extract it...

```
$(CURL) https://github.com/plus3it/tfdocs-awk/archive/master.tar.gz | tar -C .cache --strip-components 1 --wildcards '*.sh' --wildcards '*.awk' -xzvf -
```

Reply:
:tableflip: had to install `gnu-tar` to get this to work with the wildcard flag. Putting [this](https://superuser.com/questions/318809/linux-os-x-tar-incompatibility-tarballs-created-on-os-x-give-errors-when-unt/688231#688231) here for future ref.